### DISCUSSION TITLE
Share your awesome PWA project here!

### DISCUSSION THREAD

**Discussion by shadowwalker** [shadowwalker/next-pwa#206] [Show and tell]
# :wave: Hey!

While you are working on your awesome PWA project, whether you need proof of concept or some beta testers, why not share and post the link to your PWA here?

**Comment by meotimdihia** [comment]
https://comick.fun is using PWA. Thanks for the updates on the project.

**Comment by iitsuraj** [comment]


**Reply by daniel-johnsson** [reply]
These sites are not what they say they are

**Reply by faiz-ansari-afk** [reply]
Its clickbait

**Reply by alexandrutocar** [reply]
Can someone delete the links? They are potentially harmful.

**Reply by rawand-faraidun** [reply]
@shadowwalker These links are clickbait

**Reply by shadowwalker** [reply]
Deleted

**Reply by iitsuraj** [reply]
no, its not a clickbait. please check these repo.

https://github.com/iitsuraj/ytthumbnaildownloader
https://github.com/iitsuraj/qr-code

**Reply by iitsuraj** [reply]
> Deleted

Check repo

**Comment by Babatunde13** [comment]
I built a React App with a PWA feature, you can check it out here
https://babatunde-password-manager.netlify.app

**Comment by morajabi** [comment]
Used for our desktop app: https://usenoor.com 
Recently migrated form `next-offline` :)

**Reply by shadowwalker** [reply]
This is a cool startup!

**Reply by iamevansobeng** [reply]
hey @morajabi what version of nextJs did you guys use for the pwa?

**Comment by tuanla6** [comment]
I build a React + Nextjs App with PWA: [https://vietnambis.com/en/](https://vietnambis.com/en/)

**Comment by gupta-ji6** [comment]
I and @prateek3255 built [Scriptified](https://scriptified.dev/) - a weekly newsletter with insightful tips, tools, resources & more on React and JavaScript.

It's built with NextJS + TailwindCSS. Read the story of building Scriptified [here](https://ayushgupta.tech/blog/story-of-scriptified).

**Reply by shadowwalker** [reply]
Nice, keep it up!

**Comment by max-programming** [comment]
I built this app https://easypastes.tk and used this awesome package to make it a PWA

**Comment by rishimohan** [comment]
Using it on https://kizie.co, it's going to be a full-fledged Twitter app for web, next-pwa has certainly helped but need to tweak it more to the needs. Thanks for working on this, makes it easy to setup PWA for Next.js!

Here's a screenshot of it as a PWA app on Mac :)

![Untitled](https://user-images.githubusercontent.com/14031295/129105920-aef0d1f6-16e0-414b-bb4e-bcdd87703169.png)

**Reply by max-programming** [reply]
Wow! The UI looks so nice!

**Reply by rishimohan** [reply]
@max-programming Thanks :) Been building it in side to explore Next.js and design skills

**Comment by anshulagx** [comment]
Implemented PWA in www.yellowjobs.org
Thanks for making it so easy.

**Comment by maccali** [comment]
Implement PWA and TWA 

PWA -> https://apod.pictu.one/

TWA -> https://play.google.com/store/apps/details?id=one.pictu.apod.twa&hl=pt_BR&gl=US

**Comment by Ivan-Mitkov** [comment]
[**Stock valuation calculator**](https://www.howtovaluestocks.com)

![Howtovaluestocks com](https://user-images.githubusercontent.com/31875249/143725334-462c0623-e9d9-4127-8f36-527cdeca1978.png)

**Comment by kirrosh** [comment]
QR code manager PWA - [My QR](https://my-qr.vercel.app/)

**Comment by koji98** [comment]
[AniBrain](https://anibrain.ai) is now a PWA :) Thanks for making this possible!

**Comment by bjcarlson42** [comment]
Using `next-pwa` on https://www.coffeeclass.io!

**Comment by yolisses** [comment]
Hi everybody! This lib is awesome. I used on [suasplantas.com](suasplantas.com) (a site to swap plants with other users)

**Comment by angelhtml** [comment]
<img src='https://angel-resume.netlify.app/unknown-2-1-1.webp'/>

**Comment by Hazihell** [comment]
Hello there! I'm looking at your guys projects but none of them sent me the notification to install it as an app. Tried on chrome desktop and mobile. Is that an optional thing ? I am looking foward to use this module in my project, but still have this concern. I am sorry if this is not the right place to ask this. Thanks

**Reply by TriupLauro** [reply]
I'm currently trying to make a next-js app into a PWA with next-pwa but it always say "No matching service worker detected." in the application tab of the chrome dev tools. I tried a few of the above apps and they seem to have the same issue. So no prompt to install and no icon in chrome to open the prompt. 

Any Idea why this happens ? Can we find a workaround ?

**Reply by TriupLauro** [reply]
Sorry, it still seems to work on Chrome. I just needed to find a way to make service worker enable on http (i used this method : https://groups.google.com/a/chromium.org/g/security-dev/c/EYqASMdFEI0)

**Comment by awitherow** [comment]
I am adding this package to both of my main projects I am working on right now:

1. [thewolfpact.co](https://thewolfpact.co)
2. [buildleansaas.com](https://buildleansaas.com)

thanks for making the web more accessible!

**Comment by william-keller** [comment]
https://4devs.net.br is using PWA. 
Thank you guys

**Comment by BlackCubes** [comment]
Many thanks to the team for making this package. Here is my personal portfolio site using `next-pwa`: [https://www.eliastgutierrez.com](https://www.eliastgutierrez.com). And here is the repo for it, if curious: [https://github.com/BlackCubes/portfolio](https://github.com/BlackCubes/portfolio).

**Comment by adomigold** [comment]
Many thanks to the team for building this package. I use it to make this beautiful movie streaming and downloading app, you can check it here at [Adomovies.com](https://adomovies.com)

**Reply by Robokishan** [reply]
is this legal ?

**Comment by pankajparashar** [comment]
**Can I DevTools?** now has PWA support, thanks to `next-pwa`!

[canidev.tools](https://www.canidev.tools/)

![social-media-original](https://user-images.githubusercontent.com/38640616/177003122-687efa49-564d-4e8b-8cbc-addd26a0b885.png)

**Reply by rishimohan** [reply]
Nice! Is this image by any chance made using pika.style?

**Comment by darwinprayoga** [comment]
https://biofip.com

It was about self-branding based on biography NFT by web 3.0, and multiple APIs were goin to build the traffic of your profile looking for identity card to Metaverse on forward.

**Comment by lukasdotcom** [comment]
[bundesliga.lschaefer.xyz](https://bundesliga.lschaefer.xyz) or on github [github.com/lukasdotcom/Bundesliga](https://github.com/lukasdotcom/Bundesliga).

This is a self hosted fantasy Bundesliga manager.

**Comment by masakifukunishi** [comment]
[PWA List](https://www.pwalist.app/)
This site is like a PWA store.
I used this Lib to support PWA.
Thank You.

**Reply by darwinprayoga** [reply]
can u add biofip.com?

**Reply by masakifukunishi** [reply]
Hi, please login and submit via the form in the link below.
https://www.pwalist.app/submissions/create

**Comment by mateusfg7** [comment]
[**Noisekun**](https://noisekun.vercel.app/), a simple ambience sound listener, inspired on noisli.com.

<img src="https://user-images.githubusercontent.com/40613276/184353292-47b3e8ca-9434-4c3c-9a31-bb8d73be08ac.png" width="70%"><img src="https://user-images.githubusercontent.com/40613276/184353309-f4fa9870-5536-4e77-a76d-c4510f8acec1.png" width="70%">

<img src="https://user-images.githubusercontent.com/40613276/184353331-a62f5ef6-3c21-4e00-80c2-fbb3197be9b1.png" width="30%"><img src="https://user-images.githubusercontent.com/40613276/184353341-df661182-91d8-4670-b84c-1e0a4b3194d5.png" width="30%">

---

website:     _https://noisekun.vercel.app/_
repository: _https://github.com/mateusfg7/Noisekun_

**Reply by clicktodev** [reply]
Love this project.
PWA doesn't seem to be working tho.
Not getting the install prompt at all.

**Reply by Robokishan** [reply]
Hey i have created same kind of app but with pomodoro and notion support: https://www.producthunt.com/posts/pomodoro-for-notion-databases

**Reply by Robokishan** [reply]
I would love collaborate and we can do much better

**Comment by easy-bios** [comment]
www.Photoparty.live is a simple shared album where you and your friends can upload to the same album.
Just share a QR code with your friends and the can upload to the same album.
https://www.photoparty.live/

![www photoparty live_(iPhone 6_7_8)](https://user-images.githubusercontent.com/42430994/191232199-c15644a8-2254-45f4-81f1-ca30b5676a0e.png)
![www photoparty live_(iPhone 6_7_8) (1)](https://user-images.githubusercontent.com/42430994/191232223-85c0363a-c46a-4efe-832e-e077d6a01e7a.png)

**Comment by KarthikeyanM28** [comment]
https://www.fr8.in/ 
India's largest truck booking platform
We use technology to make road freight more efficient, reducing costs for transporters and increasing earnings for truck owners.
![image](https://user-images.githubusercontent.com/77260283/194483555-970d7d24-2908-47c5-9c28-6ddf46789e37.png)

**Comment by vishnupeas** [comment]
https://www.karthikaspices.com/
I made this website for a client of mine. I **love** making my websites and enabling PWA.

![image](https://user-images.githubusercontent.com/35342513/213395334-7dfd3e54-d797-411d-93d0-9099783ee3ef.png)

**Comment by josholdham** [comment]
https://www.worldiesle.com/
I made a small little Wordle-inspired game for guessing famous football (soccer) goals from a series of still images.
I'm still yet to explore more advanced features of next-pwa, but getting the basics in place was a breeze. Thanks to the team for making it such an easy package to work with!

![IMG_4649](https://user-images.githubusercontent.com/1517021/214551180-36cbb2b2-e738-4700-bc52-f46a9fce17fa.PNG)

**Comment by Robokishan** [comment]
I made this https://www.producthunt.com/posts/pomodoro-for-notion-databases app and added support for PWA using this library.

The only problem now I am having is that when the user is running the app.  and I push some code changes it gets auto-updated even though it doesn't suppose to and the user losts all his/her data

**Comment by Yagasaki7K** [comment]
Thanks for this support PWA

https://webessentials.com.br/

**Comment by tiodot** [comment]
I made an website with one quotation daily that supports PWA

https://www.icalendar.top/

**Comment by khanguyen7** [comment]
I thought that supporting PWA would be a headache, but thankfully I used Next and even more so thankfully this plugin exists. It was easy to implement and the only issues I had were because I can't spell hahaha

I don't have an actual app but for learning purposes I turned one of my old projects into a pwa: https://edmonton-lrt-ee-outage-tracker.vercel.app/

**Comment by vegancat** [comment]
https://game.landoflandless.xyz is using next-pwa to deliver a better experience for players. check it out and don't be shy! give us feedback about your experience in our socials

**Comment by floriandiud** [comment]
Hi everyone. Thanks for this great lib. I use it on [Datablist](https://www.datablist.com/), a CSV and data manipulation tool!

**Comment by scottpaulin** [comment]
A [Parquet Viewer](https://www.parquetdata.com/view) for opening and filtering Parquet files

**Comment by rirh** [comment]
# EasyReads - Download and Read Directly

![EasyReads Logo](https://tigerzh.com/easy-reads/android-chrome-512x512.png)

Discover a world of knowledge with EasyReads, the online platform dedicated to providing a vast collection of PDF format e-books. From literature and science to history and more, we offer a diverse selection of books that cater to students, professionals, and book enthusiasts alike. 

📚 **Explore Diverse Topics:** Dive into a rich variety of subjects, spanning literature, science, history, and more. No matter your interest, EasyReads has something for everyone.

🎯 **Simplify Reading:** With an intuitive interface and easy navigation, EasyReads makes reading a breeze. Whether you're a student, a professional, or simply a book lover, enjoy a seamless and joyful reading experience.

📖 **Read Anytime, Anywhere:** Access your favorite e-books whenever you want, wherever you are. EasyReads allows you to carry your library in your pocket.

Ready to embark on a journey of knowledge? Start exploring the endless treasure of wisdom with EasyReads today!

Visit our website: [https://tigerzh.com/easy-reads/](https://tigerzh.com/easy-reads/)

![EasyReads Screenshot](https://github.com/shadowwalker/next-pwa/assets/21093108/7bd5dedd-be62-4ee7-84e3-0e811c9ece4f)
![EasyReads Screenshot Light](https://github.com/shadowwalker/next-pwa/assets/21093108/7e3f9d46-6e1b-4e37-b9a6-37b6642f602b)
![EasyReads Screenshot Dark](https://github.com/shadowwalker/next-pwa/assets/21093108/54ce6b50-da2d-4001-b187-390caa17def0)


🚀 **Powered by Next.js:** EasyReads is built using Next.js, ensuring a fast, efficient, and modern reading experience.

Experience the joy of reading with EasyReads. Download, read, and explore with ease.

---

**Note:** The actual URLs, images, and formatting may need to be adjusted based on your website and design choices.

**Comment by SugamGroup** [comment]
Thank you for the above informatiom. To know more about the best Freight services in India, check this out- [Freight Forwarding ](https://www.sugamgroup.com/freight-forwarding-services-in-india/)

**Comment by MaxiMir** [comment]
Thanks for your work 

It's social network: https://2bebetter.pro/en/maximir?d=15%3A61%2C2%3A9

![image](https://github.com/shadowwalker/next-pwa/assets/30494646/dfd962bb-c8e0-41a9-bf85-27f5e27a37ce)

**Comment by MixMe** [comment]
Thanks for your work!

https://www.photolamus.com

**Comment by Quilljou** [comment]
Thanks for your work!

https://www.designsystemhunt.com

<img width="1403" alt="截屏2024-01-04 20 49 43" src="https://github.com/shadowwalker/next-pwa/assets/18414813/74ef14fa-5449-430a-bdc9-b4a38464f9b2">

**Comment by fredanup** [comment]
I want to show background web push notifications or when the app is closed is possible with next-pwa

**Comment by ps011** [comment]
Thanks a lot for your work. Here's mine: https://exchange-rates-delta.vercel.app/

**Comment by milton-imageline** [comment]
Hi, I noticed the plugin last contribution was more than two years ago. Has anyone tested it with Next 14? Is there any issue I might encounter?

**Reply by zacBkh** [reply]
Hello, I would be happy to have the answer

**Comment by williamcaamot** [comment]
Here's my project; 
https://www.ernær.no

**Comment by funwarioisii** [comment]
I'm using next-pwa in this project: https://www.fussy.fun
Thanks!

**Comment by Clems02** [comment]
Hi, can we use this lib to create a pwa or she is uptodate ? The latest update is in 2022.

**Comment by nirnejak** [comment]
I'm using next-pwa for my portfolio website https://nirnejak.com/ 🎨

**Comment by akoskm** [comment]
A minimalistic todo application https://onetaskx.com/ ✅ no sign up needed 🤗

**Comment by web3-coder-hub** [comment]
天天影视 https://ttys.app

**Comment by hubenchang0515** [comment]
https://shift.js.org/ is an online code editor based on WebAssembly that allows you to run programming languages ​​such as Python, Lua, and Ruby in your browser, and uses PWA to reduce the bandwidth overhead of WebAssembly.

![preview](https://private-user-images.githubusercontent.com/11847852/523470898-c1567e67-0a84-43d6-8606-563a8c57f560.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzM4NjM3NTcsIm5iZiI6MTc3Mzg2MzQ1NywicGF0aCI6Ii8xMTg0Nzg1Mi81MjM0NzA4OTgtYzE1NjdlNjctMGE4NC00M2Q2LTg2MDYtNTYzYThjNTdmNTYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAzMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMzE4VDE5NTA1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4NjI4YjJkMzc5NzIzOGFlZGJkMTliNDQ2Y2IzMzE2ZWJhOTA0YjM4ZDVlNGJkMTNiMjk3MzVjZTE2ODFlNTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.h4KleKvna_RZ-_B1tpvO99SonmLwbha3_SR27UrjP08)

**Comment by divyamdotfoo** [comment]
[jsdojo](https://jsdojo.vercel.app/) is an open-source, multilingual learning app for javascript.
[Github](https://github.com/divyamdotfoo/js-dojo)

<img width="1224" height="779" alt="Screenshot 2026-03-11 at 2 35 26 PM" src="https://github.com/user-attachments/assets/13885e0b-b13c-4126-bccd-70239ebba8bf" />

**Comment by Yagasaki7K** [comment]
[MeuBoi](https://meuboi.com.br/) is a cattle management platform that helps farmers track livestock, financial performance, purchases, sales, and herd profitability.

<img width="1920" height="971" alt="image" src="https://github.com/user-attachments/assets/36cdc24f-2b34-4baf-9217-65aff54fd85a" />

**Comment by ElmiraZeynalova** [comment]
[Forge](https://pwa-workout-tracker-2ymf.vercel.app/) is a lightweight PWA for logging workouts with offline-first architecture and cross-device sync
<img width="1593" height="1188" alt="preview2" src="https://github.com/user-attachments/assets/f42fcb31-41df-483e-857d-9060bb0de63c" />
